### PR TITLE
feature/rds

### PR DESCRIPTION
**Pull request by ana0**

### CODE REVIEW

File: circles-api/modules/rds/main.tf [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    Name = "terraform-example-rds-security-group"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_db_instance" "default" {
+  identifier = "${var.rds_instance_identifier}"
+  allocated_storage = "${var.allocated_storage}"
+  engine = "postgres"
+  engine_version = "9.6.3"
```
Comment (xwvvvvwx) [inline circles-api/modules/rds/main.tf]: why not use the latest version of postgres?

File: circles-api/modules/rds/main.tf [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  availability_zone = "${element(var.availability_zones, count.index)}"
+  tags {
+    Name = "rds-${element(var.availability_zones, count.index)}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_db_subnet_group" "default" {
+  name = "${var.rds_instance_identifier}-subnet-group"
+  description = "Terraform example RDS subnet group"
```
Comment (xwvvvvwx) [inline circles-api/modules/rds/main.tf]: copy paste error.

File: circles-api/notes.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+the terraform state goes in s3
```
Comment (xwvvvvwx) [inline circles-api/notes.txt]: do we wanna commit this file?

**Comment by ana0** [comment]
Addressed PR comments, except for deletion_protection .. looks like it was added to terraform only really recently (it's not in my version yet, for example) .. https://github.com/terraform-providers/terraform-provider-aws/issues/6008

Do we all want to upgrade?

**Comment by xwvvvvwx** [comment]
I'm fine with requiring the latest version of terraform.

**Comment by edzillion** [comment]
Problems `apply`ing this due to problems with CIDR block allocation.

**Pull request merged**